src/include/razermouse_driver.h

```diff
@@ -90,6 +90,9 @@
 #define RAZER_NEW_MOUSE_RECEIVER_WAIT_MIN_US 31000
 #define RAZER_NEW_MOUSE_RECEIVER_WAIT_MAX_US 31100
 
+#define RAZER_ATHERIS_RECEIVER_WAIT_MIN_US 400000
+#define RAZER_ATHERIS_RECEIVER_WAIT_MAX_US 400100
```

Comment:
You've added these properties but they don't appear to be used anywhere?

Reply:
Razer Atheris Receiver is already present in razermouse_driver.h, razermouse_driver.c, & razerdevice.c. I'm not sure what you're asking.

Reply:
You added two constants in this PR, but they aren't used anywhere. Or am I missing the place where it's used?

Reply:
Oh, I see. I'm working on adding support for the Basilisk X Hyperspeed and added the missing entries for the Atheris Receiver in that workflow. One minute. I'll add it to this branch.

Reply:
I added the missing entries to razermouse_driver.c. Please review.

Reply:
@edualm, the Basilisk V3 isn't working without this PR, so please merge it if I adequately addressed your concern.